src/container.ts

```diff
@@ -147,6 +147,7 @@ export interface ResolveOptions {
    * returns `undefined` rather than throwing an error.
    */
   allowUnregistered?: boolean
+  defaultArgs?: Map<string, any>
```

Comment:
```suggestion
  /**
   * A map of dependencies and their default value in case
   * there is no registration in the container.
   */
  defaultArgs?: Map<string, any>
```